airflow-core/src/airflow/ui/public/i18n/locales/pl/dags.json

```diff
@@ -19,7 +19,8 @@
       "active": "Aktywne",
       "all": "Wszystkie",
       "paused": "Wstrzymane"
-    }
+    },
+    "runIdPatternFilter": "Szukaj wykonań"
```

Comment:
Is it clear from the context that it's about "Dag" runs? (the word Dag appears explictly in the other translations)

Reply:
I added it, but there is a little problem with plural forms.

When it was only "wykonanie", it was obvious that it was plural. But when we add "Wykonanie Daga" - then in Polish the plural forms will depend whether it is mulitple Dag Runs  of the same Dag ("Wykonania Daga") or multiple Dag Runs of multiple Dags ("Wykonania Dagów"). And it's not clear from the context if this is one or the other case.

So I am a bit hesitant. For me "Wykonanie" should be enough and we avoid the "multiple/single Dag ambiguity". 

WDYT @mobuchowski @kacpermuda  ?